com.unity.multiplayer.mlapi/Tests/Runtime/GlobalObjectIdHash/NetworkObjectGlobalObjectIdHashTests.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using NUnit.Framework;
+using UnityEditor.PackageManager;
+using UnityEditor.SceneManagement;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+
+namespace MLAPI.RuntimeTests
+{
+    public class NetworkObjectGlobalObjectIdHashTests
+    {
+        private Scene m_TestScene;
+
+        private void OnSceneLoaded(Scene scene, LoadSceneMode mode)
+        {
+            if (scene.name == nameof(NetworkObjectGlobalObjectIdHashTests))
+            {
+                m_TestScene = scene;
+            }
+        }
+
+        [UnitySetUp]
+        public IEnumerator Setup()
+        {
+            SceneManager.sceneLoaded += OnSceneLoaded;
+
+            var execAssembly = Assembly.GetExecutingAssembly();
+            var packagePath = PackageInfo.FindForAssembly(execAssembly).assetPath;
+            var scenePath = Path.Combine(packagePath, $"Tests/Runtime/GlobalObjectIdHash/{nameof(NetworkObjectGlobalObjectIdHashTests)}.unity");
+
+            yield return EditorSceneManager.LoadSceneAsyncInPlayMode(scenePath, new LoadSceneParameters(LoadSceneMode.Additive));
```

Comment:
`EditorSceneManager.LoadSceneAsyncInPlayMode` API doesn't require the scene to be listed under Build Settings in a project. that makes it isolated from a specific Unity project which makes the package independent from a Unity project.

Reply:
There's no way to get rid of the hardcoded "GlobalObjectIdHash" and "NetworkObjectGlobalObjectIdHashTests" parts of the string ?

Reply:
`GlobalObjectIdHash` is the folder name that contains these tests.
`NetworkObjectGlobalObjectIdHashTests.unity` is the scene this test is implemented on.